### PR TITLE
refactor: remove karma tests

### PR DESCRIPTION
**Pull request by StommePoes**
@umahaea @veerabvm no karmas, no test directory, no `npm test` command anymore. I wasn't sure if the turn-SCSS-colours function in the gulpfile was only for tests or also for something else, so it's only commented out here.

### CODE REVIEW

**Comment by umahaea** [comment]
@StommePoes - guess you deleted those shell_script file run_tests.sh. We need this file to trigger the selenium(ux-test-platform). You need to restore this file and make sure the path is correct in the .travis.yml file.

File: package.json [hunk 1]
```diff
@@ -9,12 +9,10 @@
   "scripts": {
     "start": "gulp",
     "lint": "gulp lint",
-    "test": "npm run build && karma start karma.conf.js --singleRun",
     "build": "gulp scss-to-json && gulp sass",
     "build-docs": "gulp build-docs",
     "gen-changelog": "node ./npm_scripts/gen-changelog.js",
     "gh-pages": "node ./npm_scripts/gh-pages.js",
-    "ci": "npm test",
```
Comment (aaronkaka) [inline package.json]: .travis.yml is still running "npm run ci"... so do you want to remove this altogether, or have the ci script do something else? As an example, you have a lint process you are comfortable with, run that at ci?

File: package.json [hunk 2]
```diff
@@ -49,17 +47,6 @@
...
-    "karma-babel-preprocessor": "^6.0.1",
-    "karma-chai-plugins": "^0.6.1",
-    "karma-chrome-launcher": "^0.2.2",
-    "karma-firefox-launcher": "^0.1.7",
-    "karma-fixture": "^0.2.5",
-    "karma-html2js-preprocessor": "^0.1.0",
-    "karma-json-fixtures-preprocessor": "0.0.6",
-    "karma-mocha": "^0.2.1",
-    "karma-phantomjs-launcher": "^0.2.1",
-    "karma-webpack": "~1.7.0",
```
Comment (aaronkaka) [inline package.json]: you also need to remove the following dependency:
"phantomjs": "^1.9.19",

**Comment by StommePoes** [comment]
In the future maybe we'll have a linter, but I don't have anything for now. I've left a commented version in package.json and removed from the shell script for now. Nothing from Lax would pass the old linter I don't believe; new linter dies on our broken gh-pages scss files.

**Pull request merged**